js-api-spec/compile.browser.test.ts

```diff
@@ -0,0 +1,137 @@
+// Copyright 2021 Google Inc. Use of this source code is governed by an
```

Comment:
```suggestion
// Copyright 2023 Google Inc. Use of this source code is governed by an
```

Same with other new files.